src/guide/installation.md

```diff
@@ -117,6 +117,15 @@ $ yarn
 $ yarn dev
 ```
 
+Or with PNPM:
+
+```bash
+$ pnpm create vite <project-name> --template vue
```

Comment:
```suggestion
$ pnpm create vite <project-name> -- --template vue
```

This didn't work for me, testing with pnpm 6.23.1. I had to include an extra `--`, a bit like with npm 7+. The Vite docs seem to show the same thing: https://vitejs.dev/guide/#scaffolding-your-first-vite-project